adls-plugins/docs/ADLSBatchSink-batchsink.md

```diff
@@ -10,6 +10,19 @@ Properties
 
 **path** Path to directory to store output files. The path must start with `adl://`
 
+*NOTE* 
+Either of configs {keyVaultUrl,kvKeyNames} OR {clientId,refreshTokenURL,credentials} MUST be specified.
+If former information is available, later configs are NOT required. Infact, former one(keyVault) is a more secured 
+approach to avoid specifying sensitive Service Principle details in Plugin config.
+Please refer to `DevNote` for using `keyVault` approach.
+
+
+**keyVaultUrl** Provide Mircroft's KeyVault URL address from where client credentials can be fetched
```

Comment:
They are user visible so can we describe them as not variables ? Key Vault URL

Reply:
Personally, I also feel that the config names used in this Referance Tab should be consistent to what user sees in Config Tab. 
However, the actual corresponding user readable name-strings are defined in widget-file (adls-plugins/widgets/ADLSBatchSink-batchsink.json), pls check this and confirm if current changes are ok.

I used this approach as it was the current approach for other already existing config options and across all other plugins. Therefore, we would need to change this for all config variables of this plugin if decided to do so.

One of the scenarios where I find it correct is when these plugins are used to generate pipelines programmatically (we are using such cases). There, we need config variable names to be populated from code.

Pls suggest.

Reply:
Our current conventions are to assume readers of this document are users who only use the UI. This is the case for the majority of users. The names here should match whatever label is given in the widget json. By convention we use capitalized words with spaces in the widget as Nitin mentioned (Key Vault URL).

I agree that it should be consistent. That was one of several things that was to be fixed in CDAP-15544.

The actual properties expected by the backend are accessible through the REST API https://docs.cdap.io/cdap/6.0.0/en/reference-manual/http-restful-api/artifact.html#H2152

Reply:
Ok. I have made the changes for above configs as suggested.